### PR TITLE
Ogbete

### PR DESCRIPTION
**Pull request by tccanini**

### CODE REVIEW

File: Zoo/src/JosPlateau/Animals.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    animalNames [3] = "Luton";
+    animalNames [4] = "Birmingham";
+    animalNames [5] = "Newscastle";
+    animalNames [6] = "Sutherland";
+    animalNames [7] = "Manchester";
+    animalNames [8] = "LeedsBradford";
+    animalNames [8] = "RobinHood";
+
+
+            for (int i = 0; i < animalNames.length; i++) {
```
Comment (edwinchukus) [inline Zoo/src/JosPlateau/Animals.java]: Hi Tim, I am not sure what this code is doing. This code would assign "String Number 1", etc. Kindly review it and reassign it.

**Review by edwinchukus — CHANGES_REQUESTED** [review]
Kindly make changes as requested.

**Review by tccanini — COMMENTED** [review]
reviewed as advised

File: Zoo/src/JosPlateau/AnimalsRunner.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/**
+ * Created by admin-batch-a on 7/16/17.
+ */
+public class AnimalsRunner {
+
+    public static void main(String[] args) {
+
+        Animals yourList= new Animals ();
+        yourList.getMyAnimalList();
+        System.out.println(yourList);
```
Comment (edwinchukus) [inline Zoo/src/JosPlateau/AnimalsRunner.java]: You cannot print isnatnce variable on the console. I need to see your thought process in these codes. I could explain more if need be.

File: Zoo/src/JosPlateau/AnimalsRunner2.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Created by admin-batch-a on 7/16/17.
+ */
+public class AnimalsRunner2 {
+
+    public static void main(String[] args) {
+        Animals2 myScan = new Animals2();
+        myScan.getMyList();
+        return;
```
Comment (edwinchukus) [inline Zoo/src/JosPlateau/AnimalsRunner2.java]: main method does not need a return statement. Kindly review all our examples please.

**Review by edwinchukus — CHANGES_REQUESTED** [review]

**Comment by edwinchukus** [comment]
I really need you to review all our examples right from day one before making these needed changes.

**Comment by tccanini** [comment]
NG,

Kindly re review updated corrections on the above code . Please refer to Edwin's comments on each line that required amendment.

ta

File: Kaduna_Tim_Arrays/src/Control701/Main.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package Control701;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+
+        System.out.println();
```
Comment (edwinchukus) [inline Kaduna_Tim_Arrays/src/Control701/Main.java:8]: I need you to remove this sout code.

File: Zoo/src/JosPlateau/Main.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package JosPlateau;
```
Comment (edwinchukus) [inline Zoo/src/JosPlateau/Main.java:1]: Remove this piece of code!!!

**Review by edwinchukus — CHANGES_REQUESTED** [review]

**Review by edwinchukus — APPROVED** [review]

**Pull request merged**